docs/reference/json-map-format.rst

```diff
@@ -13,31 +13,35 @@ The following fields can be found in a Tiled JSON file:
 Map
 ---
 
-+-------------------+----------+----------------------------------------------+
-| Field             | Type     | Description                                  |
-+===================+==========+==============================================+
-| width             | int      | Number of tile columns                       |
-+-------------------+----------+----------------------------------------------+
-| height            | int      | Number of tile rows                          |
-+-------------------+----------+----------------------------------------------+
-| tilewidth         | int      | Map grid width.                              |
-+-------------------+----------+----------------------------------------------+
-| tileheight        | int      | Map grid height.                             |
-+-------------------+----------+----------------------------------------------+
-| orientation       | string   | Orthogonal, isometric, or staggered          |
-+-------------------+----------+----------------------------------------------+
-| layers            | array    | Array of `Layers <#layer>`__                 |
-+-------------------+----------+----------------------------------------------+
-| tilesets          | array    | Array of `Tilesets <#tileset>`__             |
-+-------------------+----------+----------------------------------------------+
-| backgroundcolor   | string   | Hex-formatted color (#RRGGBB) (Optional)     |
-+-------------------+----------+----------------------------------------------+
-| renderorder       | string   | Rendering direction (orthogonal maps only)   |
-+-------------------+----------+----------------------------------------------+
-| properties        | object   | String key-value pairs                       |
-+-------------------+----------+----------------------------------------------+
-| nextobjectid      | int      | Auto-increments for each placed object       |
-+-------------------+----------+----------------------------------------------+
++-------------------+----------+----------------------------------------------------------+
+| Field             | Type     | Description                                              |
++===================+==========+==========================================================+
+| version           | int      | The JSON format version                                  |
```

Comment:
This should be `number`. While the version is `1` currently, it will be `1.1` with the upcoming release.

Reply:
OK:)